README.md

```diff
@@ -512,6 +512,8 @@ This feature allows all of git's color-moved options to be rendered using delta
     map-styles = bold purple => syntax magenta, bold cyan => syntax blue
 ```
 
+There is a pair of features provided in [themes.config](https://github.com/chtenb/delta/blob/master/themes.gitconfig) called `zebra-dark` and `zebra-light` which utilise the moved colors by displaying them as a faint background color on the affected lines while keeping syntax highlighting as the foreground color. You can enable one of these features by stacking it upon the theme you are using.
+
```

Comment:
I'm sorry @chtenb -- I wrote this question ages ago but somehow it got left as a "Pending" review comment instead of just being posted immediately. I'll rebase this branch so that the merge conflicts are gone but I did have this one question.

Reply:
I see, no worries :) Do you have a clearer phrasing in mind, or should there be an example?

Reply:
I think an example would help. I'm not clear yet what you mean by "stacking it upon the theme you are using". `zebra-light` and `zebra-dark` set core styles such as `minus-style` and `plus-style` so it seems to me that they would mostly overwrite the styles from another theme that the user is using?

Reply:
I was thinking like `features = my-theme zebra-dark`. This way zebra dark overrides the git colors specifically but leaves the rest of your preferred theme intact. That's how I use it currently anyway.